opnode/test/system_test.go

```diff
@@ -207,12 +217,44 @@ func TestSystemE2E(t *testing.T) {
 	if !verboseGethNodes {
 		log.Root().SetHandler(log.DiscardHandler())
 	}
+
+	sequencerDBFile, err := ioutil.TempFile("", "bss.*.json")
+	require.Nil(t, err)
+	defer func() {
+		_ = sequencerDBFile.Close()
+		_ = os.Remove(sequencerDBFile.Name())
+	}()
+
 	cfg := defaultSystemConfig(t)
 
 	sys, err := cfg.start()
 	require.Nil(t, err, "Error starting up system")
 	defer sys.Close()
 
+	// Batch Submitter
+	batchSubmitter, err := bss.NewBatchSubmitter(bss.Config{
```

Comment:
We should move this to the test setup, so we run the batch submitter in all E2E tests, instead of just this one.

Reply:
Still think this needs to be done before merge. Can't remove the old batch submitter from the opnode if we don't run the new external one in every e2e test.

Reply:
I moved over both the batch submitter and the l2 output submitter to the harness so they run on every test case